internal/datacoord/server.go

```diff
@@ -805,8 +805,7 @@ func (s *Server) startFlushLoop(ctx context.Context) {
 func (s *Server) postFlush(ctx context.Context, segmentID UniqueID) error {
 	segment := s.meta.GetSegment(segmentID)
 	if segment == nil {
-		log.Warn("failed to get flused segment", zap.Int64("id", segmentID))
-		return errors.New("segment not found")
+		return errors.New("segment not found, might be a faked segemnt, ignore post flush")
```

Comment:
will print in flush loop.
Most cases we can ignore the error (because it's due to faked segment,), but just want to add a few debug info when the flush path not working

Reply:
ah nvm I didn't catch #20108 update.